public/index.html

```diff
@@ -1,13 +1,25 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie Server!</h1>
+<head>
+  <meta charset="utf-8">
+  <title>Hoodie</title>
+</head>
+<body>
+<h1>Hello from Hoodie Server!</h1>
 
-    <p>You see this because your app has no <code>public/</code> folder.</p>
-    <p>We are working on putting something more useful in here :)
-  </body>
+<p>You see this because your app has no <code>public/</code> folder.</p>
+<p> But don't worry, we're here to get you started! </p>
+<h2>Get going with Hoodie</h2>
+<p>Your first step should be creating a <code>public</code> folder inside your project folder.</p>
+<p>Next up, just create your index.html inside that public directory.</p>
+<p>We're almost here: Paste this into your index.html just before your closing tag for the body:
+  <strong> &lt;script src=hoodie/client.js"&gt;&lt;script&gt;</strong></p>
+<p>This is necessary for your own javascript files to pick up the hoodie methods.</p>
+<p>Now handle it just like you would handle your frontend project, hoodie won't get in your way. </p>
+<p>Setup your asset folders and create your .css and .js files.</p>
+<p>Happy coding :)</p>
+<h2>List of Hoodie methods</h2>
+<p>To find out more about Hoodie.store and Hoodie.account methods, please refer to <a href="https://github.com/hoodiehq/hoodie-store-client/#api">github.com/hoodiehq/hoodie-store-client</a> and <a href="https://github.com/hoodiehq/hoodie-account-client/#api">github.com/hoodiehq/hoodie-account-client</a> 
```

Comment:
Line length could be reduced here by adding line breaks so it's more readable in the source.